### DISCUSSION TITLE
2024-09-18

### DISCUSSION THREAD

**Discussion by yutingzhao1991** [ant-design/ant-design-web3#1164] [中文社区双周会]
上次双周会记录：https://github.com/ant-design/ant-design-web3/discussions/1133

**Comment by yutingzhao1991** [comment]
本次议题：
- 过一下 SIWE 的设计 https://github.com/ant-design/ant-design-web3/issues/1107
- 跟进一下钱包为空的设计稿
- 继续跟进上次双周会遗留问题 https://github.com/ant-design/ant-design-web3/discussions/1133

**Comment by thinkasany** [comment]
[#1161 ](https://github.com/ant-design/ant-design-web3/pull/1161#issuecomment-2357360222)

官网暗黑模式这个图片是不是也可以和设计师提一下

![image](https://github.com/user-attachments/assets/ee9ccc85-9dd6-4c6d-9752-a1a94e93b126)

**Reply by yutingzhao1991** [reply]
这个好像已经有了，一会对一下

**Reply by yutingzhao1991** [reply]
https://github.com/ant-design/ant-design-web3/issues/1165 替换为这三张图就可以了
